src/shared/common/components/__tests__/phase-banner.spec.js

```diff
@@ -17,4 +17,9 @@ describe('Phase Banner component', () => {
             render(<PhaseBanner feedback="http://some.domain" />)
         ).toMatchSnapshot();
     });
+    it('should display the correct test notice when NOTPROD', () => {
+        expect(
+            render(<PhaseBanner environment="NOTPROD" />)
+        ).toMatchSnapshot();
+    });
```

Comment:
Also add a test for displaying the correct message when prod

Reply:
Added explicit test for this - there was a default test but explicit is better